R/readutils.R

```diff
@@ -282,6 +282,64 @@ readbinarycf <- function(files, T=48, obs=5, Nop=1, endian="little",
   return(invisible(ret))
 }
 
+
+# read binary correlation functions sample by sample, return as a list of length "nosamples"
```

Comment:
Not that I've noticed. I find it very convenient as a return value, because one can then simply use lapply with a lambda function for the analysis. In addition, ```lapply``` can trivially be replaced by ```mclapply```, gaining automatic support for parallelisation (the main reason my "hadron.obs" analysis was based around lists of lists).  I guess the temporary object ```Cf``` could be a pre-allocated array here. The benefit of doing it with lists though, is that one gains automatic support for the case that ```nosamples * noconfs``` is not uniform over the data set (although I doubt that case would ever play a role in a real analysis..